### PR TITLE
fix(ngMock): attach `$injector` to `$rootElement`

### LINKED ISSUE #14022: ngMock - injector not attached to $rootElement
Under ngMock when injector was already created it's not connected to a $rootElement, so:
      $rootElement.injector()`
returns an undefined value.

I wish to use $rootElement to create sub-elements for test purposes, then tested piece of code can use:
     `angular.element(localElement).injector()`
to retrieve already initialised injector.

### PR DESCRIPTION
**Pull request by gkalpak**
Fixes #14022

### CODE REVIEW

**Comment by gkalpak** [comment]
I can't think of any problems that this change might create, but maybe I'm missing something...

File: test/ngMock/angular-mocksSpec.js [hunk 1]
```diff
@@ -1647,6 +1647,10 @@ describe('ngMock', function() {
     it('should create mock application root', inject(function($rootElement) {
       expect($rootElement.text()).toEqual('');
     }));
+
+    it('should attack the `$injector` to `$rootElement`', inject(function($injector, $rootElement) {
```
Comment (campersau) [inline test/ngMock/angular-mocksSpec.js]: *attack* -> *attach*

**Comment by Narretz** [comment]
LGTM (except for the typo)
It's not super important, but it's consistent with the non-mock version.

**Comment by gkalpak** [comment]
Fixed the typo and merged. Thx !
Backported to `v1.5.x` (1a94ac9)  and `v1.4.x` (fad4dc0).

**Pull request closed without merging**